### PR TITLE
[JEX-608] Use Expeditor to bump version and trigger release build

### PR DESCRIPTION
**Pull request by schisamo**
This change updates this repo to begin using the following standard
behavior from Expeditor following a merge to the master branch:

* Automatically bump the `PATCH` version in the `VERSION` file (see https://git.io/vQQYR)
* Create a git tag for the new version (see https://git.io/vQQYR)
* Trigger a release build on manhattan.ci.chef.co using the new tag (see https://git.io/vQQY1)
* Automatically update CHANGELOG (coming soon...the `built_in:update_changelog` merge action is in development)

More details on Expeditor can be found at:
https://github.com/chef/es-lita/tree/master/lita-expeditor

Signed-off-by: Seth Chisamore <schisamo@chef.io>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -185,4 +190,4 @@
 
 
 
-\* *This Change Log was automatically generated by [github_changelog_generator](https://github.com/skywinder/Github-Changelog-Generator)*
\ No newline at end of file
+\* *This Change Log was automatically generated by [github_changelog_generator](https://github.com/skywinder/Github-Changelog-Generator)*
```
Comment (tduffield) [inline CHANGELOG.md]: We can probably remove this.

File: VERSION [hunk 2]
```diff
@@ -0,0 +1 @@
+1.31.2
```
Comment (tduffield) [inline VERSION]: The latest change in the changelog is 1.31.1. Which is correct?

File: .expeditor/config.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+product_key: inspec
+
+# Slack channel in Chef Software slack to send notifications about build failures, etc
+slack:
+  notify_channel: compliance-notify
```
Comment (schisamo) [inline .expeditor/config.yml]: @adamleff Wasn't sure if you wanted this to be `#inspec-notify` or `#compliance-notify`? Which ever you choose I'll be sure to sync up where the Jenkins notifications are sent.

Comment (adamleff) [reply .expeditor/config.yml]: `#inspec-notify` please 🙂

File: VERSION [hunk 4]
```diff
@@ -0,0 +1 @@
+1.31.1
```
Comment (schisamo) [inline VERSION:1]: We'll need to ensure this reflects the current latest version on `master/HEAD` before merging this PR.

**Review by tduffield — APPROVED** [review]

**Review by adamleff — APPROVED** [review]
Let's do this.

**Pull request merged**